src/Components/Home.css

```diff
@@ -0,0 +1,87 @@
+body {
+  background-size: cover;
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.first-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.second-container {
+  width: 50%;
+  background-color: #ff550030;
+  height: 170px;
+  border-radius: 2px;
+  border: 1px solid #bad9ff23;
+  box-sizing: border-box;
+}
+
+.header {
+  height: 165px;
+  background-color: #0072ffb3;
+  border-radius: 11px 21px;
+}
+
+.header h1 {
+  font-size: 30px;
+  color: aliceblue;
+  font-weight: 600;
+  text-align: left;
+  padding: 2%;
+}
+
+.flag {
+  width: 25%;
+  height: 90px;
+  margin-right: 5%;
+  margin-left: auto;
+  margin-bottom: -60px;
+}
+
+ul {
+  padding: 0%;
+}
+
+.country-name a {
+  width: 100%;
+  text-decoration: none;
+  font-size: large;
+  font-weight: 900;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  color: aliceblue;
+  margin-right: 3px;
+  text-align: right;
+}
```

Comment:
- [ ] If you take a look at the original design, you will see that the name of the country stays at the bottom. What if you could place the country's name at the bottom and the flag stays at the top so your design closely matches the original design as required? Kindly adjust that 👍🏽

[ Optional Change ] 
- [ ] What if you could make the icon have the same link as the country's name so that even when the icon is clicked, it takes the user to the details page? 

**The Original Design**

<img width="401" alt="Screenshot 2023-03-24 at 9 08 58 AM" src="https://user-images.githubusercontent.com/50941074/227476227-02df33b5-926c-4b89-ba80-56c2526737f9.png">

**Your Design**

<img width="452" alt="Screenshot 2023-03-24 at 9 07 08 AM" src="https://user-images.githubusercontent.com/50941074/227476321-f206f3b3-6a98-43e9-b4b0-ee382df8b0e7.png">